media-plugins/mythplugins/mythplugins-31.0.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
+PYTHON_REQ_USE="xml"
+
+#BACKPORTS="03f44039848bd09444ff4baa8dc158bd61454079"
+MY_P=${P%_p*}
+
+inherit python-single-r1 readme.gentoo-r1
+
+DESCRIPTION="Official MythTV plugins"
+HOMEPAGE="https://www.mythtv.org"
+# mythtv and mythplugins are separate builds in the same github MythTV/mythtv repository
+SRC_URI="https://github.com/MythTV/mythtv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="0/${PV}"
+
+MYTHPLUGINS="mytharchive +mythbrowser mythgame \
+mythmusic mythnetvision +mythnews +mythweather mythzmserver mythzoneminder"
+IUSE="${MYTHPLUGINS} alsa cdda cdr exif fftw +hls ieee1394 libass +opengl raw +theora +vorbis +xml xvid"
+
+# Mythnetvision temporarily disabled by upstream - should be fixed soon.
+REQUIRED_USE="
+	!mythnetvision
+	mytharchive? ( ${PYTHON_REQUIRED_USE} )
+	mythnetvision? ( ${PYTHON_REQUIRED_USE} )
+	mythmusic? ( vorbis )
+	mythnews? ( mythbrowser )
+"
+RDEPEND="
+	dev-libs/glib:2
+	dev-libs/openssl:=
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtopengl:5
+	dev-qt/qtsql:5
+	media-libs/freetype:2
+	media-libs/libpng:=
+	sys-apps/util-linux
```

Comment:
util-linux is part of @system, so don't think it's needed. [See](https://devmanual.gentoo.org/general-concepts/dependencies/) "implicit system dependency".

Reply:
Removed